src/init.cpp

```diff
@@ -1401,6 +1401,8 @@ bool AppInitMain(InitInterfaces& interfaces)
         uacomments.push_back(cmt);
     }
     strSubVersion = FormatSubVersion(CLIENT_NAME, CLIENT_VERSION, uacomments);
+    strVersion = FormatVersion(CLIENT_VERSION);
+    strFullVersion = FormatFullVersion(CLIENT_NAME, CLIENT_VERSION);
```

Comment:
Remove these. No point keeping strings around globally that aren't used else where and extremely cheap to derive.

```suggestion
```